content/en/docs/reference/tools/map-crictl-dockercli.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Mapping from dockercli to crictl
+content_type: reference
+---
+
+{{<caution>}}
+This page is deprecated and will be removed in Kubernetes 1.27.
+{{</caution>}}
```

Comment:
Use a note callout here. Caution is for advice that you should heed if you don't want to put your cluster at risk (warning is for “never do this: _x_” notices).